scripts/test-runner.js

```diff
@@ -266,7 +266,7 @@ function runTest(name, code, options: PrepackOptions, args) {
   let delayUnsupportedRequires = code.includes("// delay unsupported requires");
   if (code.includes("// inline expressions")) options.inlineExpressions = true;
   if (code.includes("// do not inline expressions")) options.inlineExpressions = false;
-  if (code.includes("// omit invariants")) options.omitInvariants = true;
+  if (code.includes("// omit invariants") || args.verbose) options.omitInvariants = true;
```

Comment:
A bit surprising that "verbose" produces less output...

Reply:
verbose is probably not the right name for this option.